doc/reference/tooling/tt_cli/tt_interactive_console.rst

```diff
@@ -0,0 +1,261 @@
+.. _tt-interactive-console:
+
+tt interactive console
+======================
+
+The ``tt`` utility features an command-line console that allows executing requests
+and Lua code interactively on the connected Tarantool instances.
+It is similar to the :ref:`Tarantool interactive console <interactive_console>` with
+one key difference: the ``tt`` console allows connecting to any available instance,
+both local and remote. Additionally, it offers more flexible output formatting capabilities.
+
+.. _tt-interactive-console-enter:
+
+Entering the console
+--------------------
+
+To connect to a Tarantool instance using the ``tt`` console, run :ref:`tt connect <tt-connect>`.
+Specify the instance URI and connection options, such as the username and the password,
+in the corresponding options.
+
+..  code-block:: console
+
+    $ tt connect 192.168.10.10:3301 -u myuser -p p4$$w0rD
+       • Connecting to the instance...
+       • Connected to 192.168.10.10:3301
+
+    192.168.10.10:3301>
+
+When connecting to an instance from the same ``tt`` environment, you can use the
+``<application>:<instance>`` string instead of the URI:
+
+..  code-block:: console
+
+   $ tt connect app:storage001
+       • Connecting to the instance...
+       • Connected to app:storage001
+
+    app:storage001>
+
+To get the list of supported console commands, enter ``\help`` or ``?``.
+To quit the console, enter ``\quit`` or ``\q``.
+
+.. _tt-interactive-console-input:
+
+Console input
+-------------
+
+Similarly to the :ref:`Tarantool interactive console <interactive_console>`, the
+``tt`` console can handle Lua or SQL input. The default is Lua. To change the input
+language, run ``\set language <language>``, for example:
+
+..  code-block:: console
+
+    app:storage001> \set language sql
+    app:storage001> select * from bands where id = 1
+    ---
+    - metadata:
+      - name: id
+        type: unsigned
+      - name: band_name
+        type: string
+      - name: year
+        type: unsigned
+      rows:
+      - [1, 'Roxette', 1986]
+    ...
+
+..  code-block:: console
+
+    app:storage001> \set language lua
+    app:storage001> box.space.bands:select { 1 }
+    ---
+    - - [1, 'Roxette', 1986]
+    ...
+
+.. note::
```

Comment:
To me, this note looks more important than the info about tab-based autocompletion because it describes the second way on how to change the input. So, I'd write this note as a regular sentence and the info about autocompletion as a note. Just a suggestion, up to you.

Reply:
I'd leave this a note because it's a bit out of the page's context: it's not about working in the console, but an external way to select the language.